design-documents/message-queue/AWSMQ.md

```diff
@@ -0,0 +1,44 @@
+#### 2- AWS MQ 
+
+AWS MQ is a Message Broker based on popular Apache ActiveMQ; it supports multiple protocols for connectivity for instance AMQP, JMS, STOMP, NMS, MQTT and WebSocket. 
+
+| #    | Important Features               |
+| ---- | -------------------------------- |
+| 1    | Queues & Topics with Ordering    |
+| 2    | Transient & persistent messaging |
+| 3    | Local & distributed transactions |
+| 4    | Request/reply                    |
+| 5    | Message filtering                |
+| 6    | Scheduled messages delivery      |
+| 7    | Large message sizes              |
+
+
+
+##### High Level Architecture
+
+Since its a managed service, it provides multi zone fault tolreance and resiliancy out of the box.
+
+<img src="AWSMQArchitectureNew.png" alt="image-20190927212814114" width="50%" />
+
+
+
+##### Evaluation Table - Details
+
+Most of the features are available since Magento is also using AMQP protocol with RabbitMQ, but the protocol version is different, RabbitMQ uses 0.9 and Amazon MQ is using AMQP 1.0; so any migration would require porting of Queues from RabbitMQ to AWS MQ. There might be some changes and adjustments to the QueueInterface implementation code to accomodate the new protocol differences. AMQP 1.0 is a completely different protocol.
```

Comment:
Not sure its easiest task, related to backward compatibility. Maybe even make sense to make its as separate module. This part have to be checked